redi-connect-front/src/pages/front/login/Login.tsx

```diff
@@ -60,7 +56,8 @@ export default function Login () {
         saveAccessToken(accessToken)
         const redProfile = await fetchSaveRedProfile(accessToken)
         if (redProfile.rediLocation !== (process.env.REACT_APP_REDI_LOCATION as RediLocation)) {
-          return setIsWrongRediLocationError(true)
+          setIsWrongRediLocationError(true)
+          return purgeAllSessionData()
```

Comment:
@mzdeloitte, let's make this line only `purgeAllSessionData()`, and put `return` on the next one. This to avoid a misunderstanding that the containing function intentionally returns the return value of `purgeAllSessionData()`.

Reply:
agreed and edited accordingly